### ISSUE TITLE
CI is failing on master

### ISSUE THREAD

**Issue by olafurpg** [scalameta/metals#298]
```
tar: Exiting with failure status due to previous errors
The command "if [ "$TRAVIS_BUILD_STAGE_NAME" = 'Release'  ]; then openssl aes-256-cbc -K $encrypted_df92ba612cfd_key -iv $encrypted_df92ba612cfd_iv -in travis/secrets.tar.enc -out travis/secrets.tar -d; tar xv -C travis -f travis/secrets.tar; fi" failed and exited with 2 during .
```
https://travis-ci.org/scalameta/metals/jobs/372419635#L476

Any idea how to fix this?

**Comment by olafurpg** [comment]
This is still an issue 

https://travis-ci.org/scalameta/metals/jobs/375519783#L467
```
The command "if [ "$TRAVIS_BUILD_STAGE_NAME" = 'Release'  ]; then openssl aes-256-cbc -K $encrypted_df92ba612cfd_key -iv $encrypted_df92ba612cfd_iv -in travis/secrets.tar.enc -out travis/secrets.tar -d; tar xv -C travis -f travis/secrets.tar; fi" failed and exited with 2 during .
```

Any idea what might be wrong?

**Comment by gabro** [comment]
I'll take a look later

**Comment by gabro** [comment]
Fixed incidentally by #294
